proto/tss/v1beta1/genesis.proto

```diff
@@ -4,7 +4,12 @@ option go_package = "github.com/axelarnetwork/axelar-core/x/tss/types";
 
 import "gogoproto/gogo.proto";
 import "tss/v1beta1/params.proto";
+import "cosmos/crypto/multisig/keys.proto";
 
 option (gogoproto.goproto_getters_all) = false;
 
-message GenesisState { Params params = 1 [ (gogoproto.nullable) = false ]; }
\ No newline at end of file
+message GenesisState {
+  Params params = 1 [ (gogoproto.nullable) = false ];
+  cosmos.crypto.multisig.LegacyAminoPubKey governance_key = 2
```

Comment:
why do we put the governance key here rather than amongst the params?

Reply:
Cuz it's part of the state that's not gonna be updated by gov module.